jsonrpc/dispatcher.go

```diff
@@ -231,11 +231,45 @@ func (d *Dispatcher) RemoveFilterByWs(conn wsConn) {
 }
 
 func (d *Dispatcher) HandleWs(reqBody []byte, conn wsConn) ([]byte, error) {
+	// first try to unmarshal to batch request
+	// if there is an error try to unmarshal to single request
+	var batchReq BatchRequest
+	if err := json.Unmarshal(reqBody, &batchReq); err == nil {
+		const (
+			openSquareBracket  = 91 // [
+			closeSquareBracket = 93 // ]
+			comma              = 44 // ,
```

Comment:
Just curious could you have used strings instead of ASCII codes?

Reply:
fixed